examples/config/src/main.rs

```diff
@@ -2,14 +2,10 @@
 #![plugin(rocket_codegen)]
 
 extern crate rocket;
+extern crate config as lib;
```

Comment:
I renamed it to `lib` because it was also renamed in the tests. I can remove the renaming–no biggie!